tests/tas-elasticsearch/pom.xml

```diff
@@ -49,11 +50,35 @@
             <artifactId>spring-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.alfresco</groupId>
+            <artifactId>alfresco-elasticsearch-shared</artifactId>
+            <version>3.0.0-M1</version>
```

Comment:
What/where is this dependency?  The version number makes it sound like it's part of this repository, but then I'd expect something with a `-SNAPSHOT` on the end.

Reply:
fixed